client/fidelity/compliance-review/components/CompliantLifetime.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import moment from 'moment'
+import ng from 'core/services/ng'
+
+export default ({ item }) => {
+    const {archive_item} = item;
+    const compliantDate = moment(archive_item.extra.compliantlifetime);
+    const now = moment()
+    const {dateformat} = ng.get('config').view;
+    const compliantDateFormatted = compliantDate.format(dateformat)
+
+    let daysLeft = compliantDate.diff(now, 'days');
+    const overdue = daysLeft < 0;
+    daysLeft = overdue ? -daysLeft : daysLeft;
+    const leftWord = overdue ? 'overdue' : 'left';
```

Comment:
Strings need to be translated. I'd also use "Days overdue" and "Days remaining" for labels.

Reply:
You're right, I need to translate those labels. The naming should still be "days left" as it was specified like that in the task description.